scripts/dev_setup.sh

```diff
@@ -27,18 +30,30 @@ function usage {
   echo "Installs or updates necessary dev tools for libra/libra."
   echo "-b batch mode, no user interactions and miminal output"
   echo "-p update ${HOME}/.profile"
-  echo "-o intall operations tooling as well: helm, terraform, hadolint, yamllint, vault, docker, kubectl, python3"
+  echo "-o install operations tooling as well: helm, terraform, hadolint, yamllint, vault, docker, kubectl, python3"
+  echo "-y install Move prover tools: z3, dotnet, boogie"
+  echo "-z installs or updates Move prover tools only"
   echo "-v verbose mode"
   echo "should be called from the root folder of the libra project"
 }
 
+function add_to_profile {
+  eval "$1"
+  FOUND=$(grep -c "$1" < "${HOME}/.profile")
+  if [ "$FOUND" == "0" ]; then
+    echo "$1" >> "${HOME}"/.profile
+  fi
+}
+
 function update_path_and_profile {
   touch "${HOME}"/.profile
   mkdir -p "${HOME}"/bin
-  export PATH="${HOME}"/bin:"${HOME}"/.cargo/bin:"${PATH}"
```

Comment:
I _think_ the path needed updated in this shell for rust tooling to install on a fresh system, but the pr tests should show us this....

Reply:
Scratch that, the tests are passing.